src/SFA.DAS.EAS.Web/Views/Transfers/TransferDetails.cshtml

```diff
@@ -100,16 +91,15 @@
                 </div>
             </fieldset>
             <button type="submit" class="button" id="submit-transfer-connection">Continue</button>
-            <a class="button-link" href="@Url.Action("Index", "EmployerCommitments")" aria-label="Cancel">Cancel</a>
```

Comment:
The cancel link has been removed, but it does show in the screenshot of 988, is this correct?